src/components/dapp-staking/statistics/Withdraw.vue

```diff
@@ -106,25 +106,39 @@ export default defineComponent({
     const substrateAccounts = computed(() => store.getters['general/substrateAccounts']);
 
     const withdraw = async (): Promise<void> => {
-      const withdrawCustomExtrinsic = async () => {
-        try {
-          const fn: SubmittableExtrinsicFunction<'promise'> | undefined =
-            $api?.value?.tx.dappsStaking.withdrawUnbonded;
-          const method: SubmittableExtrinsic<'promise'> | undefined = fn && fn();
-          method && (await callFunc(method));
-        } catch (e) {
-          dispatchError((e as Error).message);
+      try {
+        const apiRef = $api.value;
+        if (!apiRef) {
```

Comment:
is there a possibility that the api is null in the action even though connection is completed on boot time? if so, it might be better to create a common pre-hook before entering the action, because it's boilerplate?

Reply:
I don't think there is a possibility that API is null. Perhaps we don't want to include the `undefined` type whenever `$api` is defined in the boot file. 

<img width="638" alt="image" src="https://user-images.githubusercontent.com/92044428/166440704-a2ce4d47-d9c7-4953-b8d7-ee86fc7512ef.png">

<img width="622" alt="image" src="https://user-images.githubusercontent.com/92044428/166440998-e4a207ce-1c67-42bd-9106-693dea69ea53.png">

Reply:
@sirius651 Fixed in this commit.
https://github.com/AstarNetwork/astar-apps/pull/378/commits/bfb8ca07541131d9f541e9df1bd54a7fef0a0951